README.rst

```diff
@@ -0,0 +1,68 @@
+===============
+pytest-kiwitcms
```

Comment:
I think I prefer to have the name to be `kiwitcms-pytest-plugin` which is more consistent with the repository URL. This will cause changes to the rest of the urls too :(